### ISSUE TITLE
No such file or directory for DISH

### ISSUE THREAD

**Issue by dhardestylewis** [mintproject/mic#305]
The file exists, however it is not detected by mic pkg run. Let me know if you need anything else to troubleshoot this error.

`
(dish) root@ffec14972422:/tmp/mint# mic pkg run
Found mic.yaml in /tmp/mint/mic/mic.yaml
MIC needs to create new directory mic/executions/09_22_20_10_56 to run the model component Do you want to continue [Y/n]: 
Create a execution directory /tmp/mint/mic/executions/09_22_20_10_56
Copying the inputs
Source: /tmp/mint/mic/data
Destination: /tmp/mint/mic/executions/09_22_20_10_56/src
Running
./run   -i1 /tmp/mint/mic/executions/09_22_20_10_56/src/gennetcdf.py -i2 /tmp/mint/mic/executions/09_22_20_10_56/src/__pycache__.zip -i3 /tmp/mint/mic/executions/09_22_20_10_56/src/era_30_-99_31_-98_dish.h5 -i4 /tmp/mint/mic/executions/09_22_20_10_56/src/na_dem_15s.nc  -o1 lo-res_png.png -o2 input_precip_data_png.png -o3 hi-res_png.png -o4 test1downscale_nc.nc -o5 anomaly_png.png -o6 output_datadownscale_png.png 
/tmp/mint/mic/executions/09_22_20_10_56/src/io.sh: line 84: era_30_-99_31_-98_dish_h5=/tmp/mint/mic/executions/09_22_20_10_56/src/era_30_-99_31_-98_dish.h5: No such file or directory
Failed
Something is wrong. You can report this problem at https://bit.ly/2zR1Tew
`

`
(dish) root@ffec14972422:/tmp/mint# ls /tmp/mint/mic/executions/09_22_20_10_56/src/
.colors.sh                 na_dem_15s.nc
dish_precip.py             output.sh
era_30_-99_31_-98_dish.h5  __pycache__.zip
gennetcdf.py               run
`

**Comment by dgarijo** [comment]
@dhardestylewis,
yes, we would need the contents of /tmp/mint (i.e., the mic yaml file, and the data and code used). If the data is too big, then a sample file would suffice.  

The `mic pkg run` command actually tests your component locally to see if the i/o has been properly organized. Have you run the previous commands before? Have you been able to run you component successfully with the trace command?

Thanks!

**Comment by dhardestylewis** [comment]
Link to the /tmp/mint (231 MB). If this is too large, I will reduce the files I provide.
https://portals-api.tacc.utexas.edu/postits/v2/e9730492-ca55-4353-ade7-e4d8e1ef3d63-010

I have first run the command successfully by itself, then run it successfully with the trace command, in the exact order from the directions:
https://mic-cli.readthedocs.io/en/latest/model_configuration/01-overview/
